### PR TITLE
Modify rule S4060: Add VB.NET code snippets

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Related to https://github.com/SonarSource/sonar-dotnet/pull/5728, cc @Corniel

### CODE REVIEW

File: rules/S4060/rspectator.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ifdef::env-github,rspecator-view[]
```
Comment (pavel-mikula-sonarsource) [inline rules/S4060/rspectator.adoc:1]: Extracting duplicate noise to a common file

File: rules/S4060/csharp/rule.adoc [hunk 2]
```diff
@@ -1,12 +1,33 @@
 include::../rule.adoc[]
-ifdef::env-github,rspecator-view[]
 
-'''
-== Implementation Specification
-(visible only on this page)
+== Noncompliant Code Example
 
-include::../message.adoc[]
+[source,csharp]
```
Comment (pavel-mikula-sonarsource) [inline rules/S4060/csharp/rule.adoc:5]: C# moved from common to C#-specific file. With `csharp` formatting. I've also removed all syntax noise that was not relevant for the rule.

File: rules/S4060/vbnet/rule.adoc [hunk 3]
```diff
@@ -1,12 +1,35 @@
...
 
-'''
-== Implementation Specification
-(visible only on this page)
+== Noncompliant Code Example
 
-include::../message.adoc[]
+[source,vbnet]
+----
+Public Class MyAttribute    ' Noncompliant
```
Comment (Corniel) [inline rules/S4060/vbnet/rule.adoc:7]: May be remove the extra spaces?

**Review by Corniel — APPROVED** [review]
Beside those extra spaces, LGTM.

File: rules/S4060/vbnet/rule.adoc (hunk 3 above)
Comment (pavel-mikula-sonarsource) [reply rules/S4060/vbnet/rule.adoc:7]: Between `MyAttribute` and the comment? It's one tab, that's fine.

**Review by Corniel — APPROVED** [review]

File: rules/S4060/vbnet/rule.adoc (hunk 3 above)
Comment (Corniel) [reply rules/S4060/vbnet/rule.adoc:7]: Why not a single space?

Comment (pavel-mikula-sonarsource) [reply rules/S4060/vbnet/rule.adoc:7]: Hard to read - too much information per square cm

**Review by mary-georgiou-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**